deploy/deployUtils.ts

```diff
@@ -245,12 +245,18 @@ export async function deployMetaswapPools(
       basePoolName!,
       metaPoolName,
     )
-    // verify contract
+    // verify contracts
     await verifyContract(hre, metaPoolName)
-  }
-  // register new pools
-  if (newDeploypools.length > 0) {
-    await registerPools(hre, newDeploypools)
+    await verifyContract(hre, lpTokenName)
```

Comment:
Note that we cant verify LP token contracts since they are minimal proxies. (We dont have the source code that matches the deployed bytecode)